playwright-e2e/pages/basePage.page.ts

```diff
@@ -29,6 +43,34 @@ export default class BasePage {
     this.locationPicker = page.locator('#location-picker');
   }
 
+  async checkYesRadioButton(): Promise<void> {
+    await this.yesRadioButton.check();
+  }
+
+  async checkNoRadioButton(): Promise<void> {
+    await this.noRadioButton.check();
+  }
+
+  async checkBritishCheckbox(): Promise<void> {
+    await this.britishCheckbox.check();
+  }
+
+  async checkIrishCheckbox(): Promise<void> {
+    await this.irishCheckbox.check();
+  }
+
```

Comment:
are basePage and nationality.page.ts both suoppsed to have checkBritishCheckbox and checkIrishCheckbox?

Reply:
nationalityPage now handles this functionality, resolving conversation